packages/odyssey-react/src/components/Button/Button.test.tsx

```diff
@@ -14,25 +14,32 @@ import React from "react";
 import { render, fireEvent } from "@testing-library/react";
 import Button from "./Button";
 
+const button = "button";
 const buttonLabel = "Button Label";
 
 describe("Button", () => {
   it("render the button", () => {
-    const { getByText } = render(<Button onClick={jest.fn()} variant="primary">{buttonLabel}</Button>);
-    
+    const { getByText } = render(<Button children={buttonLabel} />);
+
     expect(getByText(buttonLabel)).toBeInTheDocument();
   });
 
+  it('renders aria attrs via omit rest props', () => {
+    const { getByRole } = render(<Button aria-describedby="foo" children="bar" />);
```

Comment:
Based on how I have types structured currently, I would have expected a type error here complaining about `aria-describedby`

Reply:
solved: this value type is provided by the FunctionComponent generic we're using